src/constants/styleProperties.js

```diff
@@ -65,7 +65,7 @@ const styleProperties = {
 
         heightActionBar: '42px',
 
-        fontSizeExtremeLarge: '32px',
+        fontSizeUltraLarge: '38px',
```

Comment:
Sir. I would like to remind you that fontSizeExtremeLarge is used in 5 places.

Reply:
added it back